src/img-optm.cls.php

```diff
@@ -369,6 +374,17 @@ private function _append_img_queue( $meta_value, $is_ori_file = false ) {
 			$short_file_path = $this->tmp_path . $short_file_path;
 		}
 
+		// Test if need to skip image size.
+		if (!$is_ori_file) {
+			// Allow users to ignore custom sizes.
+			$disabled_sizes = apply_filters( 'litespeed_img_optim_sizes_skipped', $this->conf( Base::O_IMG_OPTM_SIZES_SKIPPED ) );
```

Comment:
1. `optim` in filter name isn't consistent w/ our current naming.
2. This filer should be done in class init. If doing here it will be low efficient.
@timotei-litespeed